tests/regression/46-apron2/79-context-insens.c

```diff
@@ -0,0 +1,46 @@
+// PARAM: --set ana.activated "['base', 'threadid', 'threadflag', 'threadreturn','mallocWrapper','mutexEvents','mutex','access','race','escape','expRelation','assert','var_eq','symb_locks','apron','memLeak']" --enable ana.int.interval_set --set ana.ctx_insens "['base', 'threadid', 'threadflag', 'threadreturn','mallocWrapper','mutexEvents','mutex','access','race','escape','expRelation','assert','var_eq','symb_locks','apron','memLeak']" --enable ana.autotune.enabled --set ana.autotune.activated "['octagon']" --set ana.specification "CHECK( init(main()), LTL(G valid-memcleanup) )" --set ana.path_sens "['mutex', 'malloc_null', 'uninit','expsplit','activeSetjmp','memLeak',      'threadflag']" --set ana.malloc.unique_address_count 5
```

Comment:
Are all these hard-coded analyses necessary for this program? It's single-threaded so many of them seem excessive.

Also, I think tests should not depend on the behavior of autotuning, which we don't give any guarantees about. Rather, tests can use annotations which were originally determined by a run with autotuning. I think the autotuner outputs what changes it makes.

Reply:
Yes, I agree! I managed to minimize it and remove the autotuner (and uncovered #1381).
The config is still somewhat exotic but I guess the circumstances where this shows are a bit niche.